packages/world/vite.config.ts

```diff
@@ -1,21 +1,7 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import { defineConfig } from "vitest/config";
-import peerDepsExternal from "rollup-plugin-peer-deps-external";
 
 /**
  * @see https://vitejs.dev/config/
  */
-export default defineConfig({
-  build: {
-    ssr: true,
-    lib: {
-      entry: ["ts/index.ts", "mud.config.ts"],
-      formats: ["es"],
-    },
-    outDir: "dist",
-    minify: true,
-    sourcemap: true,
-    target: "esnext",
-  },
-  plugins: [peerDepsExternal()] as any,
-});
+export default defineConfig({});
```

Comment:
You could just remove this file vitest doesn't need a config

Reply:
are we even using vitest here?

Reply:
yep it's unused. Vitest was only for config, which is now in its own package